sagemaker-debugger/tensorflow2/tensorflow2_keras_custom_container/tf2-keras-custom-container.ipynb

```diff
@@ -0,0 +1,319 @@
+{
```

Comment:
<p>1)Full description of 2.x support is available at {link} -- on the link we should have table saying what is supported in TF.115 | TF.2.0,TF2.1</p><p> </p><p>2) Amazon SageMaker Debugger{link to debugger github} is a new feature</p><p> </p><p>3) In this notebook, we'll show you how to use a built-in rule to monitor your training job using a tf.keras (TF 2.1.0) ResNet example.  --&gt; we 'll show how to use sagemaker-debugger and use a built-in rule {Rulename} to monitor training job in real time.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
2 and 3 -> done

1 -> we dont have public documentation about this. probably add this in smdebug docs? or the aws doc where the limitations of smdebug are mentioned